.github/workflows/continuous-integration.yml

```diff
@@ -38,15 +38,24 @@ jobs:
           - "7.4"
           - "8.0"
           - "8.1"
+          - "8.2"
         dbal-version:
           - "default"
+        extension:
+          - "pdo_sqlite"
         include:
           - php-version: "8.0"
             dbal-version: "2.13"
+            extension: "pdo_sqlite"
           - php-version: "8.1"
             dbal-version: "3@dev"
-          - php-version: "8.2"
+            extension: "pdo_sqlite"
+          - php-version: "7.4"
             dbal-version: "3@dev"
+            extension: "sqlite3"
+          - php-version: "8.1"
+            dbal-version: "3@dev"
+            extension: "sqlite3"
```

Comment:
I think one extra job for that driver should be enough. Similarly, we only have one job for mysqli (well 2: one for MySQL, one for MariaDB). Do you expect to catch failures with one job that you would not catch with the other one?

Reply:
Okay, let's remove the PHP 7.4 job then.